index.html

```diff
@@ -0,0 +1,191 @@
+<html>
+<head> 
+  <title> </title>
+ <link rel = "stylesheet" href="css/style.css" />
+</head>
+<body>
+<nav id="navbar"> 
+    <header> JS Documentation </header>
```

Comment:
[Optional]
 You can also use tag "h1" instead of tag "header". It's better for the next projects.

Reply:
- Kindly add an explanation to your comments. it might be obvious for you but not for someone else.
- Also Remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.

Reply:
[Optional] 
Kindly use the tag "h1" for the main heading instead of the tag "header". It's better for the next projects.